packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -85,9 +101,12 @@ export const EntityHeaderTable = (props: EntityHeaderTableProps) => {
     setParseErrors,
   } = useEntityHeaderTableState(references, onUpdateEntityIDsTextbox, onUpdate)
 
-  const setInvalidEntityIDError = useCallback((invalidEntityIDs: string[]) => {
-    setParseErrors([`Invalid Synapse ID(s): ${invalidEntityIDs.join(',')}`])
-  }, [])
+  const setInvalidEntityIDError = useCallback(
+    (invalidEntityIDs: string[]) => {
+      setParseErrors([`Invalid Synapse ID(s): ${invalidEntityIDs.join(',')}`])
+    },
+    [setParseErrors],
```

Comment:
ESLint complained about missing dependencies--these are all `useCallback` so it is appropriate to include all dependencies